gef.py

```diff
@@ -9943,6 +9946,13 @@ def load_plugins_from_directory(plugin_directory: pathlib.Path):
         dbg(f"Loading extra plugins from directory={directory}")
         return load_plugins_from_directory(directory)
 
+
+    def load_virtualenv(self, new_path: Optional[pathlib.Path] = None):
+        path = new_path or gef.config["gef.virtualenv_path"]
+        if path:
+            activate_script_path = pathlib.Path(path)/"bin"/"activate_this.py"
```

Comment:
Here `path` is already known to be of type `pathlib.Path`, no neeed for a 2nd allocation, 
```
    activate_script_path = path / "bin/activate_this.py"
    assert activate_script_path.is_file()
```